### ISSUE TITLE
Improve Accessibility score by adding [lang] attribute to <html> element for Users & Group page

### ISSUE THREAD

**Issue by GingLuo** [CMU-313/Teedy#68]
This page (http://localhost:8080/src/#/user) currently has an accessibility score of 95. It has the issue that the <html> element does not have a [lang] attribute, which makes the screen reader assume this page is in the default language. But the screen reader will not pronounce the webpage correctly if the page is not in the default language.

**Comment by GingLuo** [comment]
I would like to work on this please!

**Comment by GingLuo** [comment]
fixes: #71
